blackout/parserapp/services/parser.py

```diff
@@ -1,9 +1,12 @@
+import os
+from pathlib import Path
 from datetime import datetime
 from bs4 import BeautifulSoup
 import sqlite3
 from selenium import webdriver
 from selenium.webdriver.chrome.service import Service
 import time
+import json
 
 
 def start_browser():
```

Comment:
Raise an error because I don't have chrome on Linux.
Error:
`selenium.common.exceptions.WebDriverException: Message: unknown error: cannot find Chrome binary`

Reply:
Install chrome on Linux and try again